localstack/services/sqs/query_api.py

```diff
@@ -169,7 +170,13 @@ def try_call_sqs(request: Request, region: str) -> Tuple[Dict, OperationModel]:
 
     # TODO: permissions encoded in URL as AUTHPARAMS cannot be accounted for in this method, which is not a big
     #  problem yet since we generally don't enforce permissions.
-    client = aws_stack.connect_to_service("sqs", region_name=region)
+    account_id = extract_access_key_id_from_auth_header(headers)
+    client = aws_stack.connect_to_service(
+        "sqs",
+        region_name=region,
+        aws_access_key_id=account_id,
+        aws_secret_access_key=account_id,
```

Comment:
Just making sure that this is what we want, using the account id as access key id AND as secret access key. Haven't looked into how we manage access here to be honest, it just looked like a possible copy paste problem.